crates/store/re_types/definitions/rerun/blueprint/views/map.fbs

```diff
@@ -0,0 +1,12 @@
+namespace rerun.blueprint.views;
+
+/// A map view.
+///
+/// \example views/map title="Use a blueprint to create a MapView."
+//TODO(ab): add a screenshot
+table MapView (
+    "attr.rerun.view_identifier": "Map"
+) {
+    /// Configures the look and feel of the map.
+    options: rerun.blueprint.archetypes.MapOptions (order: 1000);
```

Comment:
related to comment on MapOptions - can we think of other future options that would inform a better parameter/field name here? Everything is an option, so calling this option isn't very helpful
(this is a tricky one and I don't have good answers, so if we can't come up with anything better right now let's leave it be)

Reply:
proposal: https://github.com/rerun-io/rerun/pull/6561#discussion_r1816118506